src/strongvelope/strongvelope.cpp

```diff
@@ -624,6 +624,30 @@ ProtocolHandler::ProtocolHandler(karere::Id ownHandle,
     }
 }
 
+template<typename T>
+std::vector<T> ProtocolHandler::str_to_a32(std::string data)
```

Comment:
Check if this method provides the same result than `DataTransformation::vector_to_b()` from `megachatapi_impl.cpp` (you can test by sending & receiving a node-attachment and downloading it at both sides). If they are compatible, remove the former one in pro of the latter one.

Also check if it is compatible with the analogous in the SDK, `MegaClient::str_to_a32()` (check the password hash generation in Webclient's code and/or set a new password including some UTF8 chars). If they are compatible, replace the function/s in the SDK with these ones and use them everywhere.